### DISCUSSION TITLE
Sorting of people by something else than age

### DISCUSSION THREAD

**Discussion by zouharvi** [machinetranslate/machinetranslate.org#250] [General]
As mentioned first in #248 regarding the ordering of people in `/people` (currently by age):

> also be discussed whether sorting by age is the best way for this? I understand that e.g. the "previous generation" should probably be shown at the bottom. But at the same time this doesn't feel entirely right for someone to get the spotlight just because they were born the latest.

It also feels confusing to search (without Ctrl+F) when they're not in alphabetical order (I'm not saying we should adopt this specifically).

**Comment by bittlingmayer** [comment]
We generally want it to be readable as a book.

So I'd say oldest *first* makes the most sense.

Same as we have RMBT, then SMT then NMT.

(The exception is updating content like events and calls for papers.)

**Reply by cefoo** [reply]
I think this is a good idea! What do you think, @zouharvi?

**Reply by zouharvi** [reply]
I understand the argument "readable as a book" and agree but still, this feels somewhat off. It also requires us to know (ask/find out?) about their age which seems way too personal for a wiki about MT.

**Reply by cefoo** [reply]
Yeah, I see your point regarding age. I had to more-or-less guess where to locate most of the younger people who are listed.
